cloud

```diff
@@ -1353,6 +1353,9 @@ function cloud_functions_env() {
     if [ -n "$extra_cc" ]; then
         env[KCIDB_EXTRA_CC]="$extra_cc"
     fi
+    if [ -n "$storage_bucket" ]; then
+        env[CACHE_BUCKET_NAME]="$storage_bucket"
```

Comment:
The environment variable name should start with `KCIDB_`. I.e. this one would become `KCIDB_CACHE_BUCKET_NAME`.

Reply:
Please pass the argument via a parameter **explicitly**, not via shared namespace.
Also, a better name for both the `cloud_functions_env()` parameter and the variable inside `execute_command()` would be something like `cache_bucket_name`, since `storage_bucket` is a bit too generic.

Reply:
okay